src/themes/OneUITheme/baseVariables/colors.ts

```diff
@@ -48,5 +45,10 @@ export const computedColorVariables = (baseVariables: CssVars): CssVars => {
             baseVariables['--color-background'],
             20
         ),
+        '--color-accent-25': mixTwoColors(
```

Comment:
can you explain what is the purpose of it, and why it is done differently then the other shade calculations?

Reply:
We still use `--color-accent` and `--color-accent-25` in the Gauge component, so the guys decided to keep these colors. Maybe it makes sense to create dedicated color for Gauge `--color-gauge` @ahalyha ?

Reply:
I agree if it is only needed in the Gauge, it can be defined there, and named in a more specific way.
Please also double check the function: --color-accent-25 vs the 3rd parameter.

Reply:
We need to decide about Gaudge component. Should it be themeable or not?
If it should be themeable, here https://github.com/textkernel/oneui/pull/819/files#diff-7c78d6eec49fa52a674350828c100d0b7ed967697022856f1bbd8b1c2e3d40a8L81 the color defined that way: color-accent-25. If do not, we can leave 2 specific colors colors

Reply:
@ahalyha, now it is themeable, right? so let's keep it themeable. If you are talking about generating the gradients then I think we can split gradients building logic for context and for additional colors:
```
           &__gradientStart {
            &--context_accent {
                stop-color: var(--color-gauge-light);
            }
        }

        &__gradientEnd {
            &--context_accent {
                stop-color: var(--color-gauge-dark);
            }
        }

    @each $context in ($contexts) {
        &__gradientStart {
            &--context_#{ $context } {
                stop-color: var(--color-#{$context});
            }
        }

        &__gradientEnd {
            &--context_#{ $context } {
                stop-color: var(--color-#{$context}-25);
            }
        }
    }
```